### PR TITLE
Chart: add a specific internal IP address for the ClusterIP service

### LINKED ISSUE #40676: Allow config a static ip for the pgbouncer clusterip service
### Description

Would be interesting to be able of setting a specific internal IP address for the ClusterIP service of the pgbouncer through the Helm chart.

### Use case/motivation

Specify the IP of the pgbouncer related service through the values.yml

For instance:
.....
spec:
  type: ClusterIP
  **{{- if .Values.pgbouncer.service.ip }}
  clusterIP: {{- .Values.pgbouncer.service.ip }}
  {{- end }}**
  selector:
    tier: airflow
    component: pgbouncer
    release: {{ .Release.Name }}
  ports:
    - name: pgbouncer
      protocol: TCP
      port: {{ .Values.ports.pgbouncer }}
    - name: pgbouncer-metrics
      protocol: TCP
      port: {{ .Values.ports.pgbouncerScrape }}
.....

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by alizmhdi**
## Related issue:

closes: #40676

add a specific internal IP address for the ClusterIP service in helm chart.

### CODE REVIEW

**Comment by alizademhdi** [comment]
Hello @potiuk,
can you review this please?

**Comment by potiuk** [comment]
We avoid to call specific people unless we know  they are related and interested here.

I think there are other reviewers already automatically assigned here.

if I am supposed to review every single PR here where others could do it as well - it's impossible with the amounts of PRs we have and very stressful in general - very mentally draining - so I deliberately limit the areas I am looking at. And chart is something I am not too "good at".

Also, by mentioning me you are decreasing chance that others will take a look because they think I will,  so try to avoid that. I suggest you just generally ask for review and wait.

**Comment by romsharon98** [comment]
changes looks good, can you add ut?

**Review by romsharon98 — APPROVED** [review]

File: chart/values.yaml [hunk 1]
```diff
@@ -2144,6 +2144,7 @@ pgbouncer:
 
   service:
     extraAnnotations: {}
+    ip: ~
```
Comment (jedcunningham) [inline chart/values.yaml]: What do you think about `clusterIP` instead?

Comment (alizmhdi) [reply chart/values.yaml]: good suggestion.

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**